sql/core/src/test/resources/sql-tests/inputs/ansi/interval.sql

```diff
@@ -1,17 +1 @@
---IMPORT interval.sql
-
--- the `interval` keyword can be omitted with ansi mode
-select 1 year 2 days;
-select '10-9' year to month;
-select '20 15:40:32.99899999' day to second;
-select 30 day day;
-select date'2012-01-01' - '2-2' year to month;
-select 1 month - 1 day;
-
--- malformed interval literal with ansi mode
-select 1 year to month;
-select '1' year to second;
-select 1 year '2-1' year to month;
-select (-30) day;
-select (a + 1) day;
-select 30 day day day;
\ No newline at end of file
+--IMPORT interval.sql
```

Comment:
Still need this test file?

Reply:
yea, the overflow behavior is different.

Reply:
In general, we should have a comment to explain why we need this file.